CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Chore: Set minimum quantity to 1 if dlc channel is open
```

Comment:
Won't this result in tiny DLCs that cannot be claimed on-chain?

Reply:
> Won't this result in tiny DLCs that cannot be claimed on-chain?

Aren't we considering the full amount on the CET transaction  including any potential off chain amount.

So I think we only have to care about the initial margin to be big enough for a channel but afterwards there shouldn't be a limit. 

Either party may loose so much that their payout can't be claimed anymore. But I don't think that's a problem.

Reply:
I think you are right!

I do think eventually we should track whether either side is getting too close to the dust limit, to at least inform about this.

Reply:
> I do think eventually we should track whether either side is getting too close to the dust limit, to at least inform about this.

afaik rust-dlc is taking care of this.